src/seedu/addressbook/commands/FindCommand.java

```diff
@@ -50,8 +50,12 @@ public CommandResult execute() {
         final List<ReadOnlyPerson> matchedPersons = new ArrayList<>();
```

Comment:
I think you can leave a few blank lines so that your method does not look so cluttered